apps/comps/components/user-agents/index.tsx

```diff
@@ -27,7 +27,7 @@ import { formatCompactNumber, toOrdinal } from "@/utils/format";
 import { VerificationBadge } from "../verification-badge";
 
 export default function UserAgentsSection() {
-  const { data: agentsData, isLoading } = useUserAgents();
+  const { data: agentsData, isLoading } = useUserAgents({ limit: 100 });
```

Comment:
`/user/agents` is paginated and this was only fetching 10 agents. Btw, the openapi spec and the response needs to be updated to include the pagination and sort params.